### ISSUE TITLE
Service Health Alert not displayed in Teams

### ISSUE THREAD

**Issue by mfehlhaber** [CU-CommunityApps/ct-az-common-alert-endpoint#190]
I am testing this Function using Test Action Group (preview) in Azure Portal. When I am triggering a Service Health alert, I do not see any output in Teams.

The trigger looks similar enough to your test data. `webHookUrl` is correct and works in other cases. `adaptiveCard` created looks good. When I copy the object in `content` into Adaptive Cards Designer, it is shown as valid, and displayed just fine.

Not sure why Teams has an issue with this particular alert.

**Comment by mikesprague** [comment]
You haven't really provided much info that helps in debugging this so let's start with the basics...

Have you deployed your own Function App in Azure that uses this codebase?

Have you tested your own Teams Incoming Webhook and confirmed it works?

If you can answer yes to both of those, have you configured your own Function App in Azure with the necessary environment variables shown in the README?

This project is a starting point for you to set your own endpoint up, not an endpoint that anyone can use. If you have not deployed it and set it up correctly, it will not work.

**Comment by mfehlhaber** [comment]
Yes, to all of the questions above. It basically works, but just not the Service Health Alert.

I am sorry to have bothered you. Maybe I should have read the README more carefully, but I could not deduce that this project is just an "as-is" without the intention to grow it into more general usefulness.

**Comment by mikesprague** [comment]
In this case, you have not provided any details that would help me debug your issue.

There should be log output (either in your terminal if you are running locally) or in the Function App logs in Azure that could identify the problem.

Additionally, in the `/assets` folder there is sample data that one can post to their endpoint to test it. Have you tried using `curl` or Postman (or whatever tool of your choosing) to post the example [SHA](https://github.com/CU-CommunityApps/ct-az-common-alert-endpoint/blob/main/assets/sample-data/service-health-alert.json) to your endpoint?

Bottom line is that without more details there isn't a lot of help anyone can offer you.

**Comment by mikesprague** [comment]
> I am sorry to have bothered you. Maybe I should have read the README more carefully, but I could not deduce that this project is just an "as-is" without the intention to grow it into more general usefulness.

@mfehlhaber 

Regarding growing the project... Pull Requests are welcome and encouraged. In your other two issues you loosely described behavior you would like without giving concrete implementation info or code examples (or a pull request). 

If you can better describe the features you are interested in and provide some technical info and perhaps even ideas for the implementation then they would have been more useful to the project and anyone else who might have had interest in the same features. 

A couple off the cuff sentences in an issue really aren't that useful without the additional info and context described above.
